assets/onboarding/purpose/index.jsx

```diff
@@ -3,7 +3,7 @@ import { __ } from '@wordpress/i18n';
 import { Button, CheckboxControl, TextControl } from '@wordpress/components';
 import { useState, useEffect } from '@wordpress/element';
 import { useQueryStringRouter } from '../query-string-router';
-import { useSetupWizardStep } from '../use-setup-wizard-step.js';
+import { useSetupWizardStep } from '../data/use-setup-wizard-step';
```

Comment:
@yscik I moved the `use-setup-wizard-step` to the `data` folder because it's more related to that. Does it make sense to you?

Reply:
Yup that sounds right.